src/reaction/views.py

```diff
@@ -36,3 +36,21 @@ def post(self, request):
         reaction.save()
         serializer = self.serializer_class(reaction)
         return success_response(serializer.data)
+
+
+class ReactionDelete(generics.GenericAPIView):
```

Comment:
could you follow the convention of using the word "remove" instead of "delete"? you can check that `/api/urls.py` follows this (ctrl f "delete" gives 0 results)